code/WorkInProgress/Cael_Aislinn/ShieldGen/shield_gen_external.dm

```diff
@@ -3,6 +3,8 @@
 //generates an energy field that loops around any built up area in space (is useless inside) halts movement and airflow, is blocked by walls, windows, airlocks etc
 
 /obj/machinery/shield_gen/external/New()
```

Comment:
This New() has no reason to exist if it's just a ..() call, just turn it into a definition.